docs/using-latest-daily.md

```diff
@@ -28,6 +28,7 @@ If you use [Package Source Mapping](https://learn.microsoft.com/en-us/nuget/cons
   <packageSource key="dotnet9">
     <package pattern="Aspire.*" />
     <package pattern="Microsoft.Extensions.ServiceDiscovery*" />
+    <package pattern="Microsoft.Extensions.Http.Resilience" />
```

Comment:
Are we sure all the uses of this package in the repo are using a version that's from the `dotnet9` feed? If some are using 8.0.10 still we'll need to include nuget.org and have a mapping for that too I think.

Reply:
In updating dotnet/aspire-samples to use dailies I'm finding I need a mapping for `Microsoft.Extensions.Resilience` too. Maybe we just make this `Microsoft.Extensions.*`?

Reply:
8.0.10 is out on nuget.org. Perhaps we should instead not try to guess and remove the package source mapping from our suggested nuget.config, as the actual one that the users end up using will be different depending on the things that their app is using.

Reply:
The way our instructions are now, we are only providing the mappings for the packages produced in this repo, but that is not a guarantee that you won't need more depending on your project.

Reply:
I think we should make it as clear as possible to help folks set their project up for the highest changes of success.

Reply:
The problem with adding `Microsoft.Extensions.*`  is that there may be packages that users will want to get from elsewhere. For instance, they may want to use stable bits from NuGet.org for all of .NET OOBs, except for the Aspire packages, so I wonder if we should try to instead have the least inclusive list and then the user can expand if needed. If Microsoft.Extensions.Resilience is needed, then we can add that one too.

Reply:
What's the blocker with this PR?

Reply:
I think it is just to make sure that the feeds are complete, but I just haven't had the chance to come back to this as it's been a bit hectic the last few days. If we want to merge as is, we could and I can send a follow-up later to address feedback, or otherwise I can take a look at this in a few days.

Reply:
yea lets its broken right now

Reply:
Sounds good, merging.